atlasdb-api/src/test/java/com/palantir/atlasdb/keyvalue/api/SweepResultsTest.java

```diff
@@ -64,52 +66,49 @@
 
     @Test
     public void accumulateWithNextRowInOrder() {
-        Assert.assertThat(RESULTS.accumulateWith(OTHER_RESULTS), Matchers.equalTo(COMBINED_RESULTS_AND_OTHER));
+        assertThat(RESULTS.accumulateWith(OTHER_RESULTS)).is(new HamcrestCondition<>(Matchers.equalTo(COMBINED_RESULTS_AND_OTHER)));
```

Comment:
Should we update the refactor tolling to transform `Matchers.equalTo` to `isEqualTo`?

Reply:
Yep, that's reasonable. I don't think we'll be able to support migration of all hamcrest matchers, but the most common ones should be doable.

Reply:
Added support for migrating common hamcrest matchers here: https://github.com/palantir/gradle-baseline/pull/850